test/integration/cookbooks/resolver-test/recipes/systemd_resolved.rb

```diff
@@ -0,0 +1,9 @@
+resolver_systemd_resolved_config '/etc/systemd/resolved.conf' do
+  dns ['1.1.1.1', '1.0.0.1', '2606:4700:4700::1111', '2606:4700:4700::1001']
+  fallback_dns '8.8.8.8,8.8.4.4,2001:4860:4860::8888,2001:4860:4860::8844'
+  domains 'localtest.com'
+end
+
+service 'systemd-resolved' do
```

Comment:
is this necessary? given that the cookbook doesn't try to switch from an existing resolver to systemd-resolved, the service should already be running and auto-reload when its config is changed

Reply:
Ubuntu 18.04 doesn't start the service by default (everything else does) so it's in there as a catch all so it doesn't fail the service running inspec test.